test/test-jsdoc.js

```diff
@@ -1,6 +1,93 @@
 var test = require("tape");
 require("../index");
 
+test("Check inline markdown", assert => {
+	let text = `
+* *strong*
+* __emphasis__
+* @light@              -> <code>light</code> 
+* @light@  @light@            -> <code>light</code> <code>light</code>
+* {@link Resource}     -> link?
+* @ignore(qx.*)
+`  
+  var test = qx.tool.compiler.jsdoc.Parser.parseComment(text);
+  console.log(test["@description"][0].body);
+  assert.end();
+});
+
+test("test ignore", assert => {
+let text = `
+  /* 
+ @ignore(process.*)
+ @ignore(global.*)
+ @ignore(Symbol.*)
+ @ignore(chrome.*)
+ 
+*/ 
+`
+var test = qx.tool.compiler.jsdoc.Parser.parseComment(text);
+console.log(test["@description"][0].body);
+assert.end();
+
+})
+
+test("Check markdown", assert => {
+	let text = `
+*
+* The @qx.bom.rest@ package consists of only one class: {@link Resource}.
+*
+* {@link Resource} allows to encapsulate the specifics of a REST interface.
+* Rather than requesting URLs with a specific HTTP method manually, a resource
+* representing the remote resource is instantiated and actions are invoked on this resource.
+* A resource with its actions can be configured declaratively or programmatically.
+*
+* There is also {@link qx.io.rest.Resource} which uses {@link Resource} under the hood.
+* The main differences between them are:
+*
+* * The event object available in the listeners (e.g. @success()@, @getSuccess()@ and @getError()@) is
+*   a native JavaScript object instead of a qooxdoo object ({@link qx.event.type.Rest}):
+* ** See {@link qx.io.rest.Resource} vs. {@link Resource}
+* ** @event.getId()@ => @event.id@
```

Comment:
What does the encapsulation in `@` do?

Reply:
@xxx@ found in some __index__.js covers the text into a <code>xxx</section>.

Reply:
Ok, like backticks in MarkDown or `<code>...</code>`? We should replace this eventually, but that's beyond the scope of your PR.

Reply:
You can use backticks now. It's an old - undocumented - qooxdoo feature.
Some others - like __xx__ are documented.
Just found it in your failure examples. 

http://www.qooxdoo.org/current/api/#qx.bom.rest :  The higlighted parts.